tests/test_crs/valid_uperf_resources.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         requests:
           cpu: 100m
           memory: 100Mi
-      hostnetwork: false
+      hostnetwork: true
```

Comment:
So idea is to reuse this test for hostnetwork instead of net new test that adds time?

Reply:
yes, I think it's not worth to add a new test for this so I took advantage of this one

Reply:
Makes sense to me.